stairctl/src/main/java/bio/terra/stairctl/commands/FlightCommands.java

```diff
@@ -0,0 +1,179 @@
+package bio.terra.stairctl.commands;
+
+import bio.terra.stairctl.StairwayService;
+import bio.terra.stairway.Control;
+import bio.terra.stairway.FlightStatus;
+import java.util.List;
+import java.util.Optional;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.shell.Availability;
+import org.springframework.shell.standard.ShellComponent;
+import org.springframework.shell.standard.ShellMethod;
+import org.springframework.shell.standard.ShellMethodAvailability;
+import org.springframework.shell.standard.ShellOption;
+
+@ShellComponent
+public class FlightCommands {
+  private static final Logger logger = LoggerFactory.getLogger(FlightCommands.class);
+  private final StairwayService stairwayService;
+
+  @Autowired
+  public FlightCommands(StairwayService stairwayService) {
+    this.stairwayService = stairwayService;
+  }
+
+  // -- Flight commands in alphabetical order --
+
+  // None of the flight command work if we are not connected, so list all commands here
+  // so they will not be available.
+  @ShellMethodAvailability({
+    "count flights",
+    "count owned",
+    "force fatal",
+    "force ready",
+    "get flight",
+    "list dismal",
+    "list flights",
+    "list owned"
+  })
```

Comment:
If we expected more `*Commands` that needed this OR it got much more complex, I might think about using a mixin to share this code, but for now it's probably OK to have two copies.

Reply:
You'll have to explain what you mean by `mixin` sometime.

Reply:
In this case it'd just mean an interface with a default implementation of `availabilityCheck()`; `*Command` would implement the interface and take the default impl, as opposed to inheriting from some an abstract with `availabilityCheck()`  (hopefully @wchamber will correct me if I'm abusing the term).